docs/source/basic_tutorials/consuming_tgi.md

```diff
@@ -110,20 +136,7 @@ gr.ChatInterface(
 ).queue().launch()
 ```
 
-The UI looks like this 👇
```

Comment:
This was redundant info, since you can directly see the same snapshot in the embedded gradio space.